### PR TITLE
[Docs]: update snacks in Fundamentals

### PR DESCRIPTION
**Pull request by WoLewicki**
Added missing snacks for examples in Fundamentals docs.

### CODE REVIEW

File: website/static/examples/next/focus-and-blur.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  });
+
+  React.useEffect(() => {
+    const unsubscribe = navigation.addListener('blur', () => {
+      alert('Screen was unfocused');
+      // Screen was focused
+      // Do something
+    });
+
+    return unsubscribe;
```
Comment (osdnk) [inline website/static/examples/next/focus-and-blur.js]: can you inline this?

File: website/static/examples/next/focus-and-blur.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+function ProfileScreen({ navigation }) {
+  React.useEffect(() => {
+    const unsubscribe = navigation.addListener('focus', () => {
+      alert('Screen was focused');
+      // Screen was focused
+      // Do something
+    });
+
+    return unsubscribe;
```
Comment (osdnk) [inline website/static/examples/next/focus-and-blur.js]: can you inline this?

Comment (osdnk) [reply website/static/examples/next/focus-and-blur.js]: are you sure it's correct?

**Comment by satya164** [comment]
Thanks a ton for this contribution!
I'm extremely thankful for these changes and proud of these great lines of code!

I strongly encourage you for further development and can't wait for the next great changes! Do not give up doing a good job!

😍

**Pull request merged**